StShared/StFTFontRegistry.cpp

```diff
@@ -112,6 +112,7 @@ StFTFontRegistry::StFTFontRegistry() {
 #else
     myFolders.add(stCString("/usr/share/fonts"));
     myFolders.add(stCString("/usr/local/share/fonts"));
+    myFolders.add(stCString(APP_PREFIX"/share/fonts"));
```

Comment:
Are fonts actually installed within "/app" on your system, or only sView?
There will be a dedicated patch for using fontconfig library instead of hard-coded folders which uses its own configuration file.

Reply:
In Flatpak `/usr/` is read-only and is what comes with the runtime. As I want to install new fonts (`DroidSansFallbackFull.ttf`, and the Nanum fonts), I need to install them with the app.

When you go to fontconfig, it'll just work as the fontconfig in Flatpak's runtime looks in `/app/share/fonts` (truncated output):
```sh
$ cat /etc/fonts/conf.d/50-flatpak.conf
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
<fontconfig>
        <dir>/app/share/fonts</dir>
        <!-- Then this so it is written when building the app -->
        <cachedir>/app/cache/fontconfig</cachedir>
</fontconfig>
```